release-notes/09-2022/README.md

```diff
@@ -0,0 +1,169 @@
+
+saltmaster/20220928.2
+
+saltmaster-init/20220928.2
+
+securityui/343881889607d5589ff7344df4dd2a0bd0d50649.1
+
+session-manager-service/20220927.2
+
+sl-k8s-hub/20220822.2
+
+sysmgmtevent/20220926.4
+
+systemsmanagementservice/20220926.11
+
+systemsui/20220926.8
+
+testinsightsui/20220929.22
+
+testmonitorservice/20220928.5
+
+userdata/20220927.11
+
+userservice-setup/20220928.1
+
+### Non Container/Chart Artifacts
+
+plotly-panel/1.1.2.zip
+
+systemlink-dataframe-datasource/1.2.1.zip
+
+systemlink-notebook-datasource/1.1.0.zip
+
+### 3rd Party Containers
+
+argoproj/argocli/v3.3.8
+
+argoproj/argoexec/v3.3.8
+
+argoproj/workflow-controller/v3.3.8
+
+bats/bats/1.8.0
+
+bitnami/minio/2022.9.7-debian-11-r4
+
+bitnami/mongodb/5.0.11-debian-11-r3
+
+bitnami/rabbitmq/3.10.7-debian-11-r17
+
+bitnami/redis-cluster/7.0.5-debian-11-r0
+
+bitnami/schema-registry/7.2.1-debian-11-r10
+
+jupyterhub/configurable-http-proxy/4.5.3
+
+jupyterhub/k8s-image-awaiter/1.2.0
+
+kiwigrid/k8s-sidecar/1.19.5
+
+kube-scheduler/v1.19.13
+
+pause/3.8
+
+quay.io/strimzi/kafka:0.31.0-kafka-3.2.1
+
+strimzi/operator/0.31.1
+
+## Helm Chart Breaking Changes
+
+- Top Level Helm Chart 0.7.x
+    - `podSecurityContext` has been moved out of `initialAdministratorMapping` and into the top level hierarchy of the `systemlink-value.yaml` file
+    - Removed `nbexecworker-apikey` and added `nbexec-argo-workflow-apikey`
+    - `nbexecservice` depends on a new bucket for the service: `systemlink-notebook-execution`
+    - Added configuration for `argoworkflows` and `nbexecservice`. See [getting-started/templates/systemlink-values.yaml](/getting-started/templates/systemlink-values.yaml) for an example configuration.
+    - The default PVC storage size for the MongoDB replica set for `nbexecservice` has increased to 50Gi, which causes Helm upgrade to fail. You manually patch these PVC sizes to allow for Helm upgrade complete successfully. Refer to [increase_nbexec_mongo_storage.sh](/increase_nbexec_mongo_storage.sh) for a script to patch there PVCs You may also delete the existing PVCs if losing notebook execution history is not an issue for your installation. Refer to [delete_nbexec_mongo_storage.sh](/delete_nbexec_mongo_storage.sh) for a script to delete the existing PVCs.
+- DataFrameService 0.7.x
+    - The requirement from the previous release to set `dataframeservice.schema-registry.externalKafka.auth.jaas.password` in the `systemlink-values.yaml` file or via `--set` has been resolved. This setting can be removed from  your values files.
+- Admin Helm chart 0.7.x
+    - `strimzi-kafka-operator` now requires `defaultImageRegistry` and `imagePullSecrets` to be specified in the `systemlink-admin-values.yaml` file. You must also continue to include these values in `systemlink-values.yaml`. Refer to [getting-started/templates/systemlink-admin-values.yaml](/getting-started/templates/systemlink-admin-values.yaml) for an example of this configuration for `strimzi-kafka-operator`.
+
```

Comment:
@arnoldcsorvasi is currently in discussion with Brad Hull about potentially breaking changes in the admin chart related to the previous release of argoworkflows. This may require some cluster resources to be manually deleted by a cluster admin. That is not currently tracked here.

Reply:
I am going to resolve this conversation since it doesn't appear like we will need to include this info at this time. If we end up needing to we can update the release notes in a future PR.